core/models/action.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+from django.db import models
+from core.models.incident import Incident
+from slack.models.slack_user import SlackUser
+
+class Action(models.Model):
+    details = models.TextField(blank=True, default="")
+    done = models.BooleanField(default=False)
+    incident = models.ForeignKey(Incident, on_delete=models.CASCADE)
+    user = models.ForeignKey(SlackUser, on_delete=models.CASCADE, blank=False, null=False)
```

Comment:
The idea behind the core app is that it's agnostic of things like Slack.  People have been spoken about adapting this tool for Microsoft teams, for example.  Can we provide a higher level abstraction for this?  Perhaps linking the action to a Django user, and allowing us to link slack users to them within the slack app?